src/py42/exceptions.py

```diff
@@ -56,20 +55,20 @@ class Py42HTTPError(Py42ResponseError):
 
     def __init__(self, exception, message=None):
         if not message:
-            response_content = "Response content: {}".format(exception.response.text)
-            message = u"Failure in HTTP call {}. {}".format(
+            response_content = f"Response content: {exception.response.text}"
+            message = "Failure in HTTP call {}. {}".format(
```

Comment:
If we use an `f-str` here for `message`, we won't need to call `str()` on the `exception`